src/configs/lang.ts

```diff
@@ -660,3 +660,19 @@ export const AN_ERROR_OCURRED = translate(
 );
 export const THE_SPECIFIC_ERROR_IS = translate('THE_SPECIFIC_ERROR_IS', 'The specific error is');
 export const DRAFT_PLANS = translate('DRAFT PLANS', 'Draft plans');
+
+export const COULD_NOT_LOAD_JURISDICTION_HIERARCHY = translate(
+  'COULD_NOT_LOAD_JURISDICTION_HIERARCHY',
+  'Failed to load Jurisdiction hierarchy'
+);
+
+export const STRUCTURES_COUNT = translate('STRUCTURES_COUNT', 'Structures Count');
+
+export const COULD_NOT_LOAD_JURISDICTION = translate(
+  'COULD_NOT_LOAD_JURISDICTION',
+  'Could not load jurisdiction'
+);
+export const JURISDICTION_ASSIGNMENT_SUCCESSFUL = translate(
+  'JURISDICTION_ASSIGNMENT_SUCCESSFUL',
+  'Location(s) assigned to plan successfully'
```

Comment:
Can we change this to Jurisdiction(s).  In OpenSRP lingo locations are things like houses, school etc i.e. entities that exist within a jurisdiction